code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -166,6 +166,11 @@
 	toxpwr = 2
 	taste_description = "fish"
 
+/datum/reagent/toxin/carpotoxin/on_mob_life(mob/living/carbon/M)
```

Comment:
are you sure you didn't mean to give this effect to rezadone?

Reply:
yes, it's like botox